src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/GameResults/GameResultViewModel.cs

```diff
@@ -81,6 +81,21 @@ public GameResultViewModel()
         /// </summary>
         public byte GameNumber { get; set; }
 
+        /// <summary>
+        /// Gets or sets the number of the game in the tournament
+        /// </summary>
+        public string FormatedGameNumber
+        {
+            get
+            {
+                return "G" + ((int)GameNumber).ToString();
+            }
+            set
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use the 'value' parameter in this property set accessor declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3237)